.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
+    <option name="myName" value="Project Default" />
+    <inspection_tool class="TypeAnnotation" enabled="false" level="WARNING" enabled_by_default="false" />
+  </profile>
+</component>
```

Comment:
Nope. This file is generated by intellij's GUI. Although it wounds me deeply to see github's red not-sign, I'm leaving it there and not hand-editing this file. Though perhaps I'll file a bug against jetbrains. /s